addon/components/metis/display-uri.hbs

```diff
@@ -37,16 +39,16 @@
   {{/if}}
 {{else}}
   {{#if this.localBasePath}}
-    <LinkTo
+    <AuLink
       @route="fallback"
       @model={{this.localBasePath}}
-      class="au-c-link au-u-word-break-all">
+      class="au-u-word-break-all">
       {{@uri}}
-    </LinkTo>
+    </AuLink>
   {{else}}
     <ul class="au-c-list-horizontal">
       <li class="au-c-list-horizontal__item">
-        <a href={{@uri}} class="au-c-link au-u-word-break-all">{{@uri}}</a>
+        <AuLinkExternal href={{@uri}} class="au-u-word-break-all">{{@uri}}</AuLinkExternal>
```

Comment:
This changes the behavior to open the link in a new tab, but i don't think it's a real issue for external links. Maybe we should add an option to the `AuLinkExternal` component 🤔.